src/stories/components/custom-table/row-item/item-core-unit.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { columns, CoreUnit } from '@ses/core/utils/test.utils';
+import ItemCoreUnit from './item-core-unit';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/CUTable/ItemCoreUnit',
+  component: ItemCoreUnit,
+  parameters: {
+    fullscreen: 'centered',
+    chromatic: {
+      viewports: [1194, 1280],
+      pauseAnimationAtEnd: true,
+    },
+  },
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          margin: '3em',
+          width: 'fit-content',
+        }}
+      >
+        <Story />
+      </div>
```

Comment:
This decorator is unnecessary here as it only adds more difficulty to see, match with the breakpoint widths, and match with the Figma image.

Reply:
done